BibBuddy/app/src/main/java/de/bibbuddy/BookRecyclerViewAdapter.java

```diff
@@ -94,6 +102,23 @@ public int getItemCount() {
     return bookList.size();
   }
 
+  /**
+   * This method fetches the number of items selected in the recyclerView.
+   *
+   * @return returns the selected recyclerView items.
+   */
+  public List<BookItem> getSelectedBookItems() {
+    List<BookItem> selectedItems = new ArrayList<>();
+    if (parent != null) {
+      int itemNumber = parent.getChildCount();
+      for (int i = 0; i < itemNumber; i++) {
+        if (parent.getChildAt(i).isSelected()) {
+          selectedItems.add(bookList.get(i));
+        }
+      }
+    }
+    return selectedItems;
```

Comment:
Same suggestions as in the AuthorRecyclerViewAdapter.
```suggestion
    List<BookItem> selectedItems = new ArrayList<>();
    
    if (parent != null) {
      for (int i = 0; i < parent.getChildCount(); i++) {
        if (parent.getChildAt(i).isSelected()) {
          selectedItems.add(bookList.get(i));
        }
      }
    }
    
    return selectedItems;
```

Reply:
Thanks for the suggestion, done!